query/src/sql/statements/value_source.rs

```diff
@@ -184,8 +184,15 @@ pub fn skip_to_next_row<R: BufferRead>(
             let c = buffer[it];
             reader.consume(it + 1);
 
+            if it == 0 && escaped {
```

Comment:
`(` and `)` can't be escaped.

Reply:
`escaped` just works for `'`.

Reply:
if the buffer is `\(`, this loop will continue, `balance += 1` will not executed.